### ISSUE TITLE
Public methods/fields of private super classes are not taken into account when computing the API "surface"

### ISSUE THREAD

**Issue by metlos** [revapi/revapi#10]
``` java
private class SuperClass {
   public SuperClass me;
}

public class SubClass extends SuperClass {
}
```

`SuperClass` is not reported as non-public type part of the API.

**Comment by metlos** [comment]
Note that this is still not fixed by commit https://github.com/revapi/revapi/commit/01a3e667e8854f2aab67d78ea1acc980aa306da1. This'll really need some thorough rewrite and simplification of the code.

**Comment by metlos** [comment]
Fixed in `revapi-java-0.11.0`. Finally ;)
